### DISCUSSION TITLE
GO Newbie, Take Two

### DISCUSSION THREAD

**Discussion by larspalo** [GrandOrgue/grandorgue#86] [General]
Originally posted 2013-04-07 21:30:01.739000 by mallette in GrandOrgue discussion forum on SourceForge<br><br>Well, with the help from here I got the Yamaha Electone HS-4 manuals and pedal mapped to GO just fine and relatively painlessly on Saturday at the church.  So, I&#039;ll be getting a box build up or purchased, a pair of huge speakers I am donating (Frazier Elevens, one of 136 pair built) down to the church, and a couple of amps installed, and we&#039;ll have our interim solution while we wait on the new pipe organ.  Haven&#039;t puzzled out how to get the expression pedal connected to GO yet and need to do that.  Don&#039;t think I&#039;ve seen anything in GO that directly addresses that, but must also confess I haven&#039;t really looked either.  Please let me know if there are any caveats or issues to avoid.

It may be dicier when the organ is installed, as it&#039;s my hope to be able to synchronize the GO Swell organ to the real swell organ, as I intend to balance the MIDI instrument against he real one so extended stops and resources are available.  I understand this has been done, but really don&#039;t have a source of info on any such projects or contact with anyone who has accomplished this.  

Dave

**Comment by larspalo** [comment]
Originally posted 2013-04-08 00:54:31.492000 by gkrev in GrandOrgue discussion forum on SourceForge<br><br>If the Electone HS-4 outputs the expression information through the MIDI out, 

(sadly my Johannus-built Wesley Moderato does not - I had to build a box to connect mechanically to the expression pedal), 

you should be able to right click on the expression pedal(s) in the GO visual interface, click on &quot;listen for event,&quot; move the expression pedal(s), and GO should pick it up and set itself to use it.

Just remember to save the organ before you close the program or you&#039;ll have to do it again the next time.

**Comment by larspalo** [comment]
Originally posted 2013-04-08 13:15:06.879000 by mallette in GrandOrgue discussion forum on SourceForge<br><br>Thanks, Greg.  So, it&#039;s either very simple or a PITA. :-).  

Dave

**Comment by larspalo** [comment]
Originally posted 2013-04-08 21:03:44.573000 by mallette in GrandOrgue discussion forum on SourceForge<br><br>So, I was wondering about the PC I should use.  Obviously, a bit of power is needed, but this box will only have OS and GrandOrgue on it.

Seems to me Linux is the obvious choice.  Would an I3 be adequate or an I5?  How much RAM?  RAM is cheap, but no point in using more than it can address.  Also, I&#039;ll definitely put the OS on SSD...that&#039;s a no brainer these days.

Any suggestions?

Dave

**Comment by larspalo** [comment]
Originally posted 2013-04-08 22:33:32.460000 by [Lars Palo](https://github.com/larspalo) in GrandOrgue discussion forum on SourceForge<br><br>It very much depends on what you&#039;re going to demand from the computer. How big samplesets are you going to use etc.

Personally I&#039;d nowadays get at least an i5 processor, that&#039;s good performance power for the money. RAM amount after what samplesets you intend to use but make sure it&#039;s fast. Seriously consider a 64 bit OS to make the most of it. And yes, I&#039;d recommend using Linux! :-)

Kind regards

Lars P

**Comment by larspalo** [comment]
Originally posted 2013-04-08 22:49:24.248000 by mallette in GrandOrgue discussion forum on SourceForge<br><br>Lars, pardon me for being slow, but I finally realized you wrote this beautiful piece of code.  Thank you, and God bless!

And thanks for that.  I was leaning towards I5 just to be sure, but wasn&#039;t sure it was necessary for Linux.  We&#039;ll start with the Burea set.  Once the &quot;real&quot; organ is in, I&#039;ll ask my expert Dr. George Ellis Mims to choose voicing additions to the instrument that will most enhance the Schantz.  As I am clueless in that arena I don&#039;t know what he&#039;ll choose. Remember, I am an organ groupie but do not play.  I&#039;ll likely give him a list of all the sample sets and let him choose the one he thinks will blend best with the Schantz, free or pay.  It&#039;s my assumption that all HW compatible sample sets will load to GrandOrgue.

I may want to contact you for a pay job, assuming you are amenable to same, to program a GUI with only the stops George chooses from a sample set to compliment the Schantz.  Our organist is probably better than we deserve, but I&#039;d rather he didn&#039;t have to choose willy nilly and I think he&#039;d prefer that too.

Thanks again for a great program.  I am looking forward to hearing it in the place of that nasty ol&#039; Allen very soon!



Dave

**Reply by larspalo** [reply]
Originally posted 2013-04-09 07:02:15.373000 by ggoode_sa in GrandOrgue discussion forum on SourceForge<br><br>Hi Dave,

&gt;It&#039;s my assumption that all HW compatible sample sets will load to GrandOrgue

All NON-ENCRYPTED HW sample sets can be programmed to be used by
GrandOrgue, so long as you have the permission of the sample set
producer to do so ;)

Kind regards,
GrahamG


On 4/9/13, Dave Mallette &lt;mallette@users.sf.net&gt; wrote:
&gt; Lars, pardon me for being slow, but I finally realized you wrote this
&gt; beautiful piece of code.  Thank you, and God bless!
&gt;
&gt; And thanks for that.  I was leaning towards I5 just to be sure, but wasn&#039;t
&gt; sure it was necessary for Linux.  We&#039;ll start with the Burea set.  Once the
&gt; &quot;real&quot; organ is in, I&#039;ll ask my expert Dr. George Ellis Mims to choose
&gt; voicing additions to the instrument that will most enhance the Schantz.  As
&gt; I am cluesless in that arena I don&#039;t know what he&#039;ll choose.  I&#039;ll likely
&gt; give him a list of all the sample sets and let him choose the one he thinks
&gt; will blend best with the Schantz, free or pay.  It&#039;s my assumption that all
&gt; HW compatible sample sets will load to GrandOrgue.
&gt;
&gt; I may want to contact you for a pay job, assuming you are amenable to same,
&gt; to program a GUI with only the stops George chooses from a sample set to
&gt; compliment the Schantz.  Our organist is probably better than we deserve,
&gt; but I&#039;d rather he didn&#039;t have to choose willy nilly and I think he&#039;d prefer
&gt; that too.
&gt;
&gt; Thanks again for a great program.  I am looking forward to hearing it in the
&gt; place of that nasy ol&#039; Allen very soon!
&gt;
&gt; Dave
&gt; ---
&gt; [GO Newbie, Take
&gt; Two](https://sourceforge.net/p/ourorgan/discussion/962124/thread/1a57a135/?limit=25#e6a2)
&gt;
&gt; ---
&gt;
&gt; Sent from sourceforge.net because you indicated interest in
&gt; &lt;https://sourceforge.net/p/ourorgan/discussion/962124/&gt;
&gt;
&gt; To unsubscribe from further messages, please visit
&gt; &lt;https://sourceforge.net/auth/subscriptions/&gt;

**Comment by larspalo** [comment]
Originally posted 2013-04-09 06:45:57.171000 by [Martin Koegler](https://github.com/e9925248) in GrandOrgue discussion forum on SourceForge<br><br>GO can load sample sets designed for GO or HW1 samplesets. 

For unencrypted HW2/HW3/HW4 sample sets, it is possible to create ODFs (eg. look at http://mps-net.de/orgel/gosite/ for some examples).

The ODF reference is part of our help.

**Comment by larspalo** [comment]
Originally posted 2013-04-09 13:38:41.285000 by [Lars Palo](https://github.com/larspalo) in GrandOrgue discussion forum on SourceForge<br><br>Hi!

&gt; Lars, pardon me for being slow, but I finally realized you wrote this beautiful piece of code.

Sorry, but such a statment I simply cannot say is true... My actual coding contributions have been very small to this wonderful software project. What I&#039;ve done (and continually try to do) though is to produce quite a few samplesets for GrandOrgue and hopefully stimulate the actual hard-working code producers to do the real job of improving GO to the rank of the best available free (open source) vpo software in the world. I&#039;ve also contributed by testing GO and the new features that are introduced, here everyone can help!

Many persons can make customized odfs for any type of avaliable samples that will load in GO, if you need any help with it I don&#039;t doubt many will provide it for free! It&#039;s really quite simple compared to the GrandOrgue internal c++ code.

Kind regards

Lars P

**Comment by larspalo** [comment]
Originally posted 2013-04-09 22:35:30.718000 by mallette in GrandOrgue discussion forum on SourceForge<br><br>Good work requires fair pay, Lars.  

So, I am going to purchase an I5 machine shortly and get this set up.  I considered building one up, but fact of the matter is the cost difference isn&#039;t that great and as this is a simple dedicated one software box it seems more sensible just to get one at Fry&#039;s.  With the whole project on my head I really don&#039;t have the time to assemble and test.

So, it will have Windoze on board, but I&#039;ll install and try Linux.  When I say &quot;try,&quot; I realized yesterday the one hangup may be the USB MIDI device.  While I have serveral very HQ soundcards around to choose from, none have the basic MIDI input.  The one I have works fine with Windoze but may or may not with Linux.  I&#039;ve been using the Mint distro lately, and it seems to handle most things just fine, which is a good thing as Linux command line stuff is incomprehensible to me and useless unless I have a character by character command to enter from someone.

We&#039;ll see...

Dave

**Reply by larspalo** [reply]
Originally posted 2013-04-10 03:00:46.433000 by marcofrancesco in GrandOrgue discussion forum on SourceForge<br><br>One reason why I fear Linux despite Windows giving me hell so often... and hate it (Windows ie) for what it stands for...

**Comment by larspalo** [comment]
Originally posted 2013-04-10 16:06:20.564000 by mallette in GrandOrgue discussion forum on SourceForge<br><br>I hear you, Marco.  I&#039;ll probably set it up with Windows and get it working, then install Mint as dual boot to test for issues. 

As the machine will have only the purpose of serving as MIDI organ I&#039;d like to have it boot straight to the organ interface.  I suspect an I5 running Linux from an SSD could do that in less than 20 seconds. 

Dave

**Comment by larspalo** [comment]
Originally posted 2013-04-10 17:58:07.484000 by wehtam721 in GrandOrgue discussion forum on SourceForge<br><br>The computer I run GrandOrgue on has an SSD and I believe that the less than 20 second boot/load time would be reasonable (depending to some extent on your settings and the size of the digital organ you&#039;d be loading).  Also, I&#039;d be happy to help you write the ODF for the instrument displaying only the stops that you&#039;d need and I&#039;m sure, as Lars said, that there would be many others willing to help also.

Matt

**Comment by larspalo** [comment]
Originally posted 2013-04-10 20:08:59.474000 by mallette in GrandOrgue discussion forum on SourceForge<br><br>Thanks, Matt.  I am going to get the box together shortly as we need to retire the ailing Allen ASAP.  However, it may be a matter of months before we install the pipe organ as we&#039;re still working on configuration in our space, some mods to the instrument prior to dismantling and moving, and such.  Even with a small instrument, these things take time to do right.

**Comment by larspalo** [comment]
Originally posted 2013-04-10 22:33:39.014000 by wehtam721 in GrandOrgue discussion forum on SourceForge<br><br>Of course.  I understand that it takes time to get to that point.  I just thought I would let you know.  It&#039;s a standing offer when your project gets to that point.

**Comment by larspalo** [comment]
Originally posted 2013-04-11 14:28:10.035000 by [Lars Palo](https://github.com/larspalo) in GrandOrgue discussion forum on SourceForge<br><br>Do remember to think about a few things before making the installations. When doing the MIDI connection think about it&#039;s connection to the computer that will run (GO)... USB is easy, but don&#039;t use multiple USB devices unless you want/enjoy the hassle of needing to reconfigure them to the sampleset manuals more or less every time you reboot - that&#039;s an USB issue that affects both windows and Linux.

On the same subject, seriously consider having one (and only one!) audio device in the computer so that it won&#039;t change device number at reboots.

It&#039;s possible to use tricks like blacklisting etc in order to a certain degree make sure that the numbering will stay the same, but to have a rock-solid reliable vpo that the organist won&#039;t need to bother about (except to play) one should make sure to eliminate all possible forseeable issues that can arise...

In short, you don&#039;t want to need to reset the MIDI configuration or the audio configuration every time you reboot the computer. For home usage this is not critical, but for a serious instrument installation such things are very important.

Kind regards

Lars P

**Comment by larspalo** [comment]
Originally posted 2013-04-15 16:26:58.355000 by mallette in GrandOrgue discussion forum on SourceForge<br><br>Thanks Lars and all.  I am now researching the benefits and perils of the hybrid instrument.  

While there are several, seems the worst is synchronizing the MIDI component to the pipe organ action.  Most solutions seem pretty pricey, but these seem to date back to 09 or earlier and I&#039;m rather hopeing there are software solutions by now.  Our board will be the Artisan board and, apparently, it has some software for several purposes though I don&#039;t know for sure if synchronicity is one of them.  I am awaiting a call back from a person from Artisan who should be able to answer that question.  The guy with whom I spoke really didn&#039;t know that part, but he said that they&#039;d done several hybrids and he knew of at least one case where a well regarded and experienced organist played one and did not know it was hybrid as the stops and such were all wired to the pipe organ console and there was no &quot;virtual&quot; screen.  That sounds promising, but I need hard information to act on.
Dave
